src/components/ui/Dropdown.tsx

```diff
@@ -0,0 +1,107 @@
+import { cva } from "class-variance-authority";
+import { useEffect, useRef, useState } from "react";
+
+const dropdownMenuStyles = cva(
```

Comment:
We can keep the style variables in a separate file

Reply:
Our other component stylings also like this, if we need to do that we should open a new issue about that. I think this is fine right now.